engine/src/main/java/org/destinationsol/assets/Assets.java

```diff
@@ -205,6 +208,50 @@ public static DSTexture getDSTexture(String path) {
         return getAtlasRegion(path, Texture.TextureFilter.Nearest);
     }
 
+    public static Animation<TextureAtlas.AtlasRegion> getAnimation(TextureAtlas.AtlasRegion original, String path) {
```

Comment:
Can we somehow change this method to accept only a path, that then internally generates the AtlasRegion if needed? Or is there some good reason to have it this way, that I'm missing?